spectacles/runner.py

```diff
@@ -552,13 +565,21 @@ async def validate_content(
                 name=self.project,
                 filters=filters,
                 include_all_explores=True,
+                get_full_project=get_full_project,
```

Comment:
Where does `get_full_project` get defined if the `if` block on line 543 isn't entered?